app/server/errors.handling.js

```diff
@@ -8,13 +8,14 @@ let render404 = function(response) {
     response.end(JSON.stringify({message:'not found'}));   
 };
 module.exports = function(data, response) {
+    var next = { then: function(callback) { callback(); }};
     if (data.error || data == '503:SERVICE UNAVAILABLE') {
         render503(response);
-        return { then: function() {} };
+        next = { then: function() {} };
```

Comment:
I like this style better than returns.  That reminds me I need to fix that in some of the frontend code from today